front/src/applications/stdcm/utils.ts

```diff
@@ -0,0 +1,36 @@
+function generateRandomString(length: number): string {
+  return [...Array(length)].map(() => Math.floor(Math.random() * 10)).join('');
+}
+
+// TODO: The number must be calculated from a hash of stdcm inputs (to have a stable number). It is currently generated randomly, so there could be duplicates.
+export function generateCodeNumber(): string {
+  const currentDate = new Date();
+  const year = currentDate.getFullYear().toString().substr(-2);
+  const month = (currentDate.getMonth() + 1).toString().padStart(2, '0');
+  const randomPart1 = generateRandomString(3);
+  const randomPart2 = generateRandomString(3);
+  return `${month}${year}-${randomPart1}-${randomPart2}`;
+}
+
+// This function displays the date as : "DD/MM/YY à HH:MM" in french, and "YY/MM/DD at HH:MM" in english
```

Comment:
This function only returns the date part (day, month, year, hours and minutes), I think the comment is for the function in utils/date

Reply:
The comment is not useful, I removed it